user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper/StepperTutorial.md

```diff
@@ -0,0 +1,91 @@
+
+The package will create an "Incident" [DOM definition](xref:DomDefinition) and some [DOM instances](xref:DomInstance) using it.
+
+> [!TIP]
+> Deploying a package is very similar to deploying a DataMiner connector. See [Deploying a DataMiner connector to your system](xref:Deploying_A_DataMiner_Connector_to_your_system).
+
+## Step 2: Create the components
+
+Now that you have deployed the package with the data, you can create the app to visualize this data.
+
+1. [Create a new application](xref:Creating_custom_apps).
+
+1. Click the pencil icon next to the initial page of the application in order to edit it.
+
+1. Hover the mouse pointer over the bar with the "+" on the left until it expands to show the Visualizations pane, and add a [stepper](xref:DashboardStepper), a [form](xref:DashboardForm), and a [table](xref:DashboardTable) component by dragging them onto the page.
+
+   You can find the stepper and form components in the *General* category and the table in the *Tables* category.
+
+The end result of this step should look like this:
+
+![Components](~/user-guide/images/StepperComponents.png)
+
+## Step 3: Add the data
+
+Add data to components by [applying a data feed](xref:Configuring_dashboard_components#applying-a-data-feed).
+
+Each of the components on the page has to receive some data:
+
+- Add the DOM definition to the stepper component and the form component.
+
+- Create a query to fetch all object manager instances of the DOM definition. See [Creating a GQI query](xref:Creating_GQI_query).
+
+- Link the *Object manager instances* feed from the table component to both the stepper and the form component.
+
+The end result of this step should look like this:
+
+![Components with their data](~/user-guide/images/StepperData.png)
+
+## Step 4: Add a save action
+
+To complete the app, you should now add a header bar with a button to save changes to incidents:
+
+1. Enable the header bar and add a button to it. See [Configuring the header bar of a low-code app page](xref:LowCodeApps_header_config).
+
+1. Add a chain of the following three actions to the *On click* event of the button:
+
+   1. Save the current changes of the form component.
+
+   1. Fetch the data in the table component.
+
+   1. Select the updated incident in the table component.
+
+   ![Action configuration](~/user-guide/images/StepperActions.png)
+
+## Step 5: Style the stepper
+
+By default, every stepper component uses the same template, but you can switch to a different template of your choice.
+
+To do so, use the *Appearance* setting in the component layout tab on the right.
+
+![Appearance presets](~/user-guide/images/StepperAppearance.png)
+
+## Step 6: Use the low-code app
+
+Now that everything is configured, you can publish and use your app.
+
+You can create new incidents using the form component and the save button in the header bar. You can inspect and update existing incidents by selecting them in the table component at the bottom.
```

Comment:
Could you make it more clear in the description of this final step what the purpose of the stepper component is in the app?